katello/katello-backup

```diff
@@ -123,6 +123,16 @@ else
     '/var/lib/puppet/ssl'
   ]
 
+  puts "Generating metadata ... "
+  metadata = File.new 'metadata', 'w'
+  metadata.puts `cat /etc/redhat-release`
+  metadata.puts "Plugin list: "
+  plugin_list = `foreman-rake plugin:list | grep 'Foreman plugin: '`.lines
```

Comment:
Right? I looked for so long to find a way... @stbenjam was the man with the answers! Hero of my week!